tst/Carnival/CarnivalTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Tests\Carnival;
+
+use Illuminate\Support\Str;
+use Mockery;
+use Mockery\Exception\InvalidCountException;
+use PHPUnit\Framework\TestCase;
+
+class CarnivalTest extends TestCase {
+    protected function tearDown () : void {
+        parent::tearDown();
+
+        if (class_exists('Mockery')) {
+            if ($container = Mockery::getContainer()) {
+                $this->addToAssertionCount($container->mockery_getExpectationCount());
+            }
+
+            try {
+                Mockery::close();
+            } catch (InvalidCountException $e) {
```

Comment:
Laravel caught the exception because under certain circumstances (the string comparisons) it wanted to ignore it. Since we never want to ignore it, why catch and immediately rethrow it?

Reply:
Okay, I will change this to do nothing if an exception is found.

Reply:
No wait lol. We don't want to catch it at all, that was my point.
Laravel only didn't rethrow it under two certain circumstances, otherwise they rethrew it. If it weren't for those two circumstances, they would never catch it at all

Reply:
Oh, sorry for misunderstanding. I can remove the try-catch all together.